content/docs/user-guide/troubleshooting.md

```diff
@@ -53,3 +53,21 @@ Unable to detect supported link types, as the
 [cache directory](/doc/command-reference/config#cache) doesn't exist. It is
 usually created automatically by DVC commands that need it, but you can create
 it manually (e.g. `mkdir .dvc/cache`) to enable this check.
+
+## Lock issue on network file system {#lock-issue}
```

Comment:
Let's make it general (and we might be able to simplify the ERROR message in DVC itself). I would mention here that in most cases it is still about `.dvc/tmp/lock` file and about another DVC process running. After that mention about filesystems like Lustre.

The first paragraph we should start with a brief mention of those two.

Reply:
Are you suggesting we can just put lock error message in dvc as "Unable to acquire lock. Aborted command execution. For troubleshooting, please visit - <link>" and make troubleshooting entry more general for possible fixes?

Reply:
@harsh8398 yes, exactly!. I'm still not sure about the `Aborted command execution` - if we should say something like ... `Most likely another DVC process is running or was terminated abruptly. Check the page ....` ...

Reply:
@shcheklein I have updated the doc. I'll add the link in the error message in DVC itself once I'll be able to resolve the test case issue. Thanks.